sw/device/tests/entropy_src_csrng_test.c

```diff
@@ -243,6 +238,7 @@ static void edn_configure(const dif_edn_t *edn, irq_flag_id_t irq_flag_id,
  */
 static void test_edn_cmd_done(const dif_edn_seed_material_t *seed_material) {
   CHECK_STATUS_OK(entropy_testutils_stop_all());
+  CHECK_DIF_OK(dif_csrng_clear_recoverable_alerts(&csrng));
```

Comment:
Do we now what recoverable alerts are been triggered?

Reply:
Yes, I fixed it in https://github.com/lowRISC/opentitan/pull/22525 but the SiVal ROM_EXT is signed so at least this avoid releasing a new signed ROM_EXT just for that.

Reply:
Thanks!

Does this somehow also help with this interrupt https://github.com/lowRISC/opentitan/issues/20747? (I suspect this is a separate issue).

Reply:
It could be related, I'll add comment there.